sos/plugins/openstack_cinder.py

```diff
@@ -27,8 +27,10 @@ class OpenStackCinder(Plugin):
     plugin_name = "openstack_cinder"
     profiles = ('openstack', 'openstack_controller')
 
-    option_list = [("db", "gathers openstack cinder db version", "slow",
-                    False)]
+    option_list = [
+        ("db", "gathers openstack cinder db version", "slow", False),
+        ("exclude", "Ignore files matching this path spec", "", "")
+    ]
```

Comment:
> remember that this data is used principally for troubleshooting and problem solving

Having two copies of the same information doesn't really help troubleshooting efforts.
In can hurt though: if the duplicate information fills up /tmp and the command errors out (which is what we're seeing).

Can you suggest an alternative way to do this that fits with sos's conventions? Thanks.